config/dockerfiles/Dockerfile.Rust

```diff
@@ -0,0 +1,27 @@
+FROM debian:jessie
```

Comment:
We don't have a policy per se, but I don't think we want to be responsible for building docker images. I'd personally rather not use a language unless there's a community image, and this pains me because I really like Rust. But i don't want that maintenance burden and overhead on a volunteer project unless there's a demonstrated stakeholder need.